packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -5,19 +5,19 @@ category: management
 
 # User Types API
 
-The User Types API provides operations to manage User Types. To manage the schemas associated with the User Types, refer to the [User Schema API reference](/docs/reference/api/schemas/#user-schema-operations).
+The User Types API provides operations to manage User Types. See [User Schema API reference](/docs/reference/api/schemas/#user-schema-operations) for information on managing the schemas associated with the User Types.
 
-## Getting Started
+## Get started
 
 Explore the User Types API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/7f16d9b7259fdf0f91f6)
 
-## User Types Operations
```

Comment:
Is it sentence case if Operations is the name of this section?

Reply:
yes, it should be User Types operations